<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUIPanel</class>
 <widget class="QWidget" name="GUIPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>227</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GUIPanel</string>
  </property>
  <widget class="QPushButton" name="runMQTTButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Inicio</string>
   </property>
  </widget>
  <widget class="QLabel" name="statusLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>200</y>
     <width>331</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Detenido</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>190</y>
     <width>64</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Estado:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>MQTT Broker</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>37</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Topic</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leHost">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>164</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>localhost</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="topic">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>35</y>
     <width>164</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>/rpi/ADC</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>241</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLCDNumber" name="lcdCh1">
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLCDNumber" name="lcdCh2">
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLCDNumber" name="lcdCh3">
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLCDNumber" name="lcdCh4">
      <property name="smallDecimalPoint">
       <bool>false</bool>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>80</y>
     <width>81</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Frecuencia</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="SpinBox">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>100</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <double>0.500000000000000</double>
   </property>
   <property name="maximum">
    <double>50.000000000000000</double>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>runMQTTButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
